examples/forward-proxy/upstream-cert/upstream.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvwIBADANBgkqhkiG9w0BAQEFAASCBKkwggSlAgEAAoIBAQDE5pVWHIECIcA9
+BiKN6LZjg7qAl/BAediRWOHtzksE96Y3pMtQcI8qQcExu4cbrJyc0O2C/wvsyhi0
+giAhMPLJlFjBcpIOhapHnc1ZIgYW/mIzPp+lS0HBrZj6SJef/83DIkou1tv8DOnD
+trUHH2o1FtB/PTkGlfh2oyNe75ZjG82DU8ToKIvJLmhc094XQIes7qRnbd/miBwL
+yvnOFeQoHQzcuaX1Of2cQ4VJlNsQwxFLdaN/Xd1zCMLL0t/XhQvKS8ofQFd9ySDb
+zfuB59NV5aUatM3hcNbLieUMPHCisyvuO36O2ZpnsAcN5aMaHZ4xAHTzpfuSwfUo
+tFnoTpdhAgMBAAECggEAECZ3pk4FOOcKzRaVO/2UBDqYUBbmUmuMG3PmaCqNFbW4
+cQT31ju9SnKFRa6DiP4SBHzQlneUA1KaTygDOaW5rFYaJ5fdqgpFqsUjREXZqQYp
+iNmAn766AIhcLV9J7Io/VtuXqRrfhV+Lxw1T8OcfP4aFCIK9jDA95oFAE/fj5nIL
+Bw+w5SUL96q7czYta8zOjJCX8AIhsx7aScR2xYmSu7qyOS6er3+O+6MiAuj9rUD6
+j7EG2bV115TDth4RU5g32U/x0EqhQh63KhncXi96LlMK+NMicmD8LytcA8uzg4W1
+G50BTF/TCfCl7FKAMnxVdi2fvZF23om16v3SWQ6N9wKBgQDfxXVg2m63qp1cLaYL
+yCZsH2C24IeNJaEm6Scy4Lv42rCC+Vv06r272uoHxobdaeFkfWQiBeYT6FmTvlSm
+CIBxg3Dh0esNYSN9l4k3WDCj0UPWSqexvGsVa5nnRCLPlBUVfNg30Jf+tBdCKiDs
+1JjdNI1aFd9pfH6jlxtK2geB+wKBgQDhQmT6E0z0uzbqIty8eYYNrUPNQdelM5nP
+1yUAYoMsCDr9GFrE3tnENeDyvC3MlooOfxSlnAcbkjpBFXTrCB3MU928Ku2KFQdM
+yoqz68/64TmqPoJAMehqkthKNb4tZJE/N5dhcR/Mt6LGy8sp/0vYp4hrmk6oX+ir
+YIKu9RbpUwKBgQDOUDgQvFQoct3zBwN5CWijd6qr8ggIUxFp1r2VYy7sGnVzzITE
+I0ob0ZLE6H2f1udAx3tzAw1GfYtZdLqG+n+yKQYCrpCRHWajfPu+U0lFQMPS57IF
+ZTs13KdhWqjdyiJGVXzkuSXqaBCpqD4DHWBzUuigjloV3/rCyacYND2H0QKBgQCc
+g2Vl3MM7yFj/lfmJe2aJWjoiA/2x64iTwQkUm15I2e8irCllYPIuQsFp5xZrcK28
+JNBTwXWu0otnbWEvbRRHRH5sn+csajAISzLUkKSRzKwkfgpCUGsMobtm1LCvGZFS
+O24TZmaR94ZCWpdvNtOcVcuT+ENu+jTP3t7ucZzZwQKBgQCaHYZAV83R08nd0Mil
+GKmV2MkFbWsoDCf1Z6I9h877Dpi+ilT5aX20Fv/eMpfUXCXwkciV2Yk9gtDmVJdR
+emAt/f7ldcVZ1dymTZIoKTVQCXexZliISq88ZwlmHbphF8LHC/0awC4XOvsDFHmg
+iAa0ukVxqPWNmpGp4wuRoa+Tpg==
+-----END PRIVATE KEY-----
```

Comment:
We're adding a private key to repo? (I'm guessing it's for test purposes but want to make sure it's by accident)

Reply:
it is a self generated cert key used to deploy testing env. Not shipped in the apicast image. It will expire in 10 years.